media/src/objects/Curve.svelte

```diff
@@ -406,52 +388,50 @@
                 type="text"
                 bind:value={params.x}
                 on:change={() => {
-            onUpdate();
-            updateCurve();
-            }}
+                    onUpdate();
```

Comment:
Yeah - thanks for fixing the indentation here. My svelte plugin doesn't auto-indent these properly (https://github.com/leafOfTree/svelte-mode/issues/11).